packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -258,6 +259,34 @@ describe('quantic-generated-answer', () => {
           });
         });
 
+        describe('when the generated answer is still streaming', () => {
+          const streamId = crypto.randomUUID();
+
+          const testMessagePayload = {
+            payloadType: 'genqa.messageType',
+            payload: JSON.stringify({
+              textDelta: testText,
+            }),
+          };
+
+          beforeEach(() => {
+            mockSearchWithGeneratedAnswer(streamId, param.useCase);
+            mockStreamResponse(streamId, testMessagePayload);
+            visitGeneratedAnswer({useCase: param.useCase});
+          });
+
+          it('should display the correct message and the streaming cursor', () => {
+            Expect.displayGeneratedAnswerCard(true);
+            Expect.generatedAnswerContains(testText);
+            Expect.generatedAnswerIsStreaming(true);
+            Expect.displayRephraseButtons(false);
+            Expect.displayLikeButton(false);
+            Expect.displayDislikeButton(false);
+            Expect.displayCopyToClipboardButton(false);
+            Expect.displayDisclaimer(false);
```

Comment:
I think we should have something like: `Expect.displayToggleButton(false);` to make sure it is not displayed.

Reply:
Yea I thought about it and decided to not add it, but since you're asking about it I'll add it.